src/main/kotlin/org/codefreak/codefreak/util/TarUtil.kt

```diff
@@ -79,28 +77,44 @@ object TarUtil {
     zip.finish()
   }
 
+  /**
+   * Check if given path name is the name of a root file like "/", "./", "." or even
+   * weird combinations like "../../."
+   */
+  private fun isRootFilename(path: String) = path.trim('/', '.').isBlank()
```

Comment:
why not `normalizeEntryName(path) == ''` ?

Reply:
`normalizeEntryName` does not handle e.g. `/` or `../`. I could modify it to
```kotlin
fun normalizeEntryName(name: String) = name.trimStart('.', '/')
```
But not sure if this is always bc

Reply:
> ```kotlin
> fun normalizeEntryName(name: String) = name.trimStart('.', '/')
> ```
sounds good